digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/RoadAddressServiceSpec.scala

```diff
@@ -486,77 +486,76 @@ class RoadAddressServiceSpec extends FunSuite with Matchers{
     // Test search by street name
     var result = roadAddressService.getSearchResults(Option("nuolirinne"))
     result.size should be(1)
-    result(0).contains("street") should be(true)
+    result.head.contains("street") should be(true)
 
     // Test search by road number
     result = roadAddressService.getSearchResults(Option("1"))
-    result.size should be(3)
-    result(2).get("road").get(0).asInstanceOf[RoadAddress].roadNumber should be(1)
+    result.size should be(2)
+    result(1)("road").head.asInstanceOf[RoadAddress].roadNumber should be(1)
```

Comment:
(just asking... miks long->string muuttaa näitä sizejä?)

Reply:
Koska testin käyttämän funktio on täytyny muuttaa